ctapipe/instrument/camera.py

```diff
@@ -33,19 +33,35 @@
 }
 
 
+_geometry_cache = {}
```

Comment:
But there are 2 `_geometry_cache = {}`, one global on line 36, and one as an attribute of CameraGeometry on line 60, are both required?